src/style.css

```
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:nth-child(n+2),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4) {
  background-image: url(./images/star-active.svg);
}
```
you may group selectors like this

---

The code looks much neater and clearer now. Feels like significant style improvement) 
Thank you!